.coveragerc

```diff
@@ -1,5 +1,5 @@
 [report]
 omit =
     */__init__.py
-    */ireporter_db.py 
```

Comment:
It is okay to omit the database file from the test coverage.

Reply:
> It is okay to omit the database file from the test coverage.

Thanks for the review,  I removed it from that file because the test coverage covers files in the app directory, it seemed like double omission wasn't necessary!